cumulusci/tasks/bulkdata.py

```diff
@@ -797,6 +799,313 @@ def _sqlite_dump(self):
                 f.write(line + "\n")
 
 
+def represent_ordereddict(dumper, data):
+    value = []
+
+    for item_key, item_value in data.items():
+        node_key = dumper.represent_data(item_key)
+        node_value = dumper.represent_data(item_value)
+
+        value.append((node_key, node_value))
+
+    return yaml.nodes.MappingNode(u"tag:yaml.org,2002:map", value)
+
+
+yaml.SafeDumper.add_representer(OrderedDict, represent_ordereddict)
```

Comment:
I don't like that importing this module has the side effect of changing the global yaml dumper.

For the inverse operation (making sure that yaml is loading into OrderedDicts in python 2) we already have `cumulusci.core.utils.ordered_yaml_load` which uses its own subclass of yaml.SafeLoader. I think I would prefer adding an `ordered_yaml_dump` function there which similarly subclasses yaml.SafeDumper.

(Soon when we drop Python 2 support we should be able to get rid of all this and just use normal dicts internally since they are ordered in Python 3.6+)